### ISSUE TITLE
fix STB merge script

### ISSUE THREAD

**Issue by judgelord** [judgelord/correspondence#33]
When this is done, make a STB clean script, then open an issue to apply codebook to STB.

**Comment by judgelord** [comment]
Do we only have data from 2015-2017? What all needs to be merged here?

**Comment by judgelord** [comment]
@rochelle-snyder Do we only have data from 2015-2017?

**Comment by rochelle-snyder** [comment]
Yes, according to the FOIA sheet. The files are on dropbox here: https://www.dropbox.com/sh/54dvbldi05s5dcj/AABy7my8QduxP9DRx96oJUuLa?dl=0
